samples/sample_fei/include/predictors_repacking.h

```diff
@@ -125,29 +125,6 @@ inline mfxI16 get4Median(mfxExtFeiPreEncMV::mfxExtFeiPreEncMVMB* preencMB, mfxI1
     return (tmpBuf[1] + tmpBuf[2]) / 2;
 }
 
-/* repackPreenc2Enc passes only one predictor (median of provided preenc MVs) because we dont have distortions to choose 4 best possible */
-
-static mfxStatus repackPreenc2Enc(mfxExtFeiPreEncMV::mfxExtFeiPreEncMVMB *preencMVoutMB, mfxExtFeiEncMVPredictors::mfxExtFeiEncMVPredictorsMB *EncMVPredMB, mfxU32 NumMB, mfxI16 *tmpBuf)
```

Comment:
no, just remove ```static``` from declaration

Reply:
I updated the commit. First, I tried just to remove the `static ` but Linker gave the `multiple definition` error, so I implement `repackPreenc2Enc` and `GetBestSetsByDistortion` functions in `fei_streamout_pak_repacking.cpp`